scripts/release-all-packages.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+source ./scripts/helpers.sh
+
+releasePackages=()
```

Comment:
i know im not a good example but maybe in bash files we could add some describing comments here and there since its not JS and we dont have any unit tests. 

like

// Gather all packages with releasePackage equals true in package.json